code/modules/food_and_drinks/machinery/smartfridge.dm

```diff
@@ -277,6 +277,14 @@
 
 /obj/machinery/smartfridge/drying_rack/on_deconstruction()
 	new /obj/item/stack/sheet/mineral/wood(drop_location(), 10)
+
+	//remove all component parts inherited from smartfridge cause they were not required in crafting
+	var/obj/item/circuitboard/machine/smartfridge/board = locate(/obj/item/circuitboard/machine/smartfridge/) in component_parts
+	component_parts -= board
+	board.moveToNullspace()
+	qdel(board)
```

Comment:
doesn't qdeling it move it to nullspace? why do we gotta be explicit here. this really shouldn't be necessary at all because this sort of thing causes wack issues in `/obj/item` destroy stuff (see #70638)

Reply:
tbf you probably won't see anything as yonked as the PR I linked since you clean out the reference to it in the `component_parts` list before you do anything but i'm just curious why you chose this pattern

Reply:
It's only because I heard somewhere that if you move an item to null space then it doesn't call destroy() and makes cleanup faster but now I realize I had it the other way around.  Thanks I'll remove that